### ISSUE TITLE
Black Minimap video output

### ISSUE THREAD

**Issue by Slokh** [enzanki-ars/rocket-league-minimap-generator#34]
Likely something to do with FFMPEG? Running on Windows.

```PTS 8589933074, next:278400000 invalid dropping st:0
DTS 8589933075, next:278440000 st:0 invalid dropping
PTS 8589933075, next:278440000 invalid dropping st:0
DTS 8589933076, next:278480000 st:0 invalid dropping
PTS 8589933076, next:278480000 invalid dropping st:0
DTS 8589933077, next:278520000 st:0 invalid dropping
PTS 8589933077, next:278520000 invalid dropping st:0
DTS 8589933078, next:278560000 st:0 invalid dropping
PTS 8589933078, next:278560000 invalid dropping st:0
DTS 8589933079, next:278600000 st:0 invalid dropping
PTS 8589933079, next:278600000 invalid dropping st:0
DTS 8589933080, next:278640000 st:0 invalid dropping
PTS 8589933080, next:278640000 invalid dropping st:0
DTS 8589933081, next:278680000 st:0 invalid dropping
PTS 8589933081, next:278680000 invalid dropping st:0
DTS 8589933082, next:278720000 st:0 invalid dropping
PTS 8589933082, next:278720000 invalid dropping st:0
DTS 8589933083, next:278760000 st:0 invalid dropping
PTS 8589933083, next:278760000 invalid dropping st:0
DTS 8589933083, next:278800000 st:0 invalid dropping
PTS 8589933083, next:278800000 invalid dropping st:0
DTS 8589933084, next:278840000 st:0 invalid dropping
PTS 8589933084, next:278840000 invalid dropping st:0
DTS 8589933085, next:278880000 st:0 invalid dropping
PTS 8589933085, next:278880000 invalid dropping st:0
DTS 8589933086, next:278920000 st:0 invalid dropping
PTS 8589933086, next:278920000 invalid dropping st:0
DTS 8589933087, next:278960000 st:0 invalid dropping
PTS 8589933087, next:278960000 invalid dropping st:0
DTS 8589933088, next:279000000 st:0 invalid dropping
PTS 8589933088, next:279000000 invalid dropping st:0
DTS 8589933089, next:279040000 st:0 invalid dropping
PTS 8589933089, next:279040000 invalid dropping st:0
DTS 8589933090, next:279080000 st:0 invalid dropping
PTS 8589933090, next:279080000 invalid dropping st:0
DTS 8589933091, next:279120000 st:0 invalid dropping
PTS 8589933091, next:279120000 invalid dropping st:0
[concat @ 00000271fddcaa60] DTS -1500 < 8589933091 out of order
frame= 8375 fps=131 q=-1.0 Lsize=     299kB time=00:04:39.06 bitrate=   8.8kbits/s dup=1396 drop=1575 speed=4.36x
video:200kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: 49.438225%
[libx264 @ 00000271fde01120] frame I:34    Avg QP: 2.09  size:   115
[libx264 @ 00000271fde01120] frame P:2110  Avg QP: 5.06  size:    30
[libx264 @ 00000271fde01120] frame B:6231  Avg QP: 8.67  size:    22
[libx264 @ 00000271fde01120] consecutive B-frames:  0.8%  0.0%  0.0% 99.2%
[libx264 @ 00000271fde01120] mb I  I16..4: 100.0%  0.0%  0.0%
[libx264 @ 00000271fde01120] mb P  I16..4:  0.0%  0.0%  0.0%  P16..4:  0.0%  0.0%  0.0%  0.0%  0.0%    skip:100.0%
[libx264 @ 00000271fde01120] mb B  I16..4:  0.0%  0.0%  0.0%  B16..8:  0.0%  0.0%  0.0%  direct: 0.0%  skip:100.0%
[libx264 @ 00000271fde01120] 8x8 transform intra:0.0%
[libx264 @ 00000271fde01120] coded y,uvDC,uvAC intra: 0.0% 0.0% 0.0% inter: 0.0% 0.0% 0.0%
[libx264 @ 00000271fde01120] i16 v,h,dc,p: 98%  0%  2%  0%
[libx264 @ 00000271fde01120] i8c dc,h,v,p: 100%  0%  0%  0%
[libx264 @ 00000271fde01120] Weighted P-Frames: Y:0.0% UV:0.0%
[libx264 @ 00000271fde01120] kb/s:5.85
```

**Comment by enzanki-ars** [comment]
Could you provide the output of `ffmepg --version` and possibly the log from the entire `rocketleaguereplayanalysis` run?  Also, was the `rocketleaguereplayanalysis` tool run from the same folder is was run from?

**Comment by Slokh** [comment]
ffmpeg version:
`ffmpeg version git-2017-12-23-d02289c`

The whole log is just this part repeating (which is for every frame I assume)
```
DTS 8589933075, next:278440000 st:0 invalid dropping
PTS 8589933075, next:278440000 invalid dropping st:0
DTS 8589933076, next:278480000 st:0 invalid dropping
PTS 8589933076, next:278480000 invalid dropping st:0
DTS 8589933077, next:278520000 st:0 invalid dropping
PTS 8589933077, next:278520000 invalid dropping st:0
DTS 8589933078, next:278560000 st:0 invalid dropping
```

**Comment by enzanki-ars** [comment]
In terms of the log, I meant to clarify that I would like to see the log for the code I wrote and the header potion of the ffmpeg log. I am curious what ffmpeg saw before starting the render, as I suspect it went wrong there missing some part of the input.

**Comment by Slokh** [comment]
Here's the log header:

```
ffmpeg version git-2017-12-23-d02289c Copyright (c) 2000-2017 the FFmpeg developers
  built with gcc 7.2.0 (GCC)
  configuration: --enable-gpl --enable-version3 --enable-sdl2 --enable-bzlib --enable-fontconfig --enable-gnutls --enable-iconv --enable-libass --enable-libbluray --enable-libfreetype --enable-libmp3lame --enable-libopencore-amrnb --enable-libopencore-amrwb --enable-libopenjpeg --enable-libopus --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libtheora --enable-libtwolame --enable-libvpx --enable-libwavpack --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxml2 --enable-libzimg --enable-lzma --enable-zlib --enable-gmp --enable-libvidstab --enable-libvorbis --enable-libvo-amrwbenc --enable-libmysofa --enable-libspeex --enable-amf --enable-cuda --enable-cuvid --enable-d3d11va --enable-nvenc --enable-dxva2 --enable-avisynth --enable-libmfx
  libavutil      56.  6.100 / 56.  6.100
  libavcodec     58.  8.100 / 58.  8.100
  libavformat    58.  3.100 / 58.  3.100
  libavdevice    58.  0.100 / 58.  0.100
  libavfilter     7.  7.100 /  7.  7.100
  libswscale      5.  0.101 /  5.  0.101
  libswresample   3.  0.101 /  3.  0.101
  libpostproc    55.  0.100 / 55.  0.100
Input #0, concat, from 'renders\rep\minimap-frames.txt':
  Duration: N/A, start: 0.000000, bitrate: N/A
    Stream #0:0: Video: png, rgb24(pc), 500x650, 25 tbr, 25 tbn, 25 tbc
Stream mapping:
  Stream #0:0 -> #0:0 (png (native) -> h264 (libx264))
Press [q] to stop, [?] for help
DTS 8589925784, next:40000 st:0 invalid dropping
PTS 8589925784, next:40000 invalid dropping st:0
DTS 8589925785, next:80000 st:0 invalid dropping
PTS 8589925785, next:80000 invalid dropping st:0
DTS 8589925786, next:120000 st:0 invalid dropping
PTS 8589925786, next:120000 invalid dropping st:0
DTS 8589925787, next:160000 st:0 invalid dropping
PTS 8589925787, next:160000 invalid dropping st:0
[libx264 @ 000001da228f1120] using cpu capabilities: MMX2 SSE2Fast SSSE3 SSE4.2 AVX
[libx264 @ 000001da228f1120] profile High, level 3.0
[libx264 @ 000001da228f1120] 264 - core 152 r2851 ba24899 - H.264/MPEG-4 AVC codec - Copyleft 2003-2017 - http://www.videolan.org/x264.html - options: cabac=1 ref=3 deblock=1:0:0 analyse=0x3:0x113 me=hex subme=7 psy=1 psy_rd=1.00:0.00 mixed_ref=1 me_range=16 chroma_me=1 trellis=1 8x8dct=1 cqm=0 deadzone=21,11 fast_pskip=1 chroma_qp_offset=-2 threads=6 lookahead_threads=1 sliced_threads=0 nr=0 decimate=1 interlaced=0 bluray_compat=0 constrained_intra=0 bframes=3 b_pyramid=2 b_adapt=1 b_bias=0 direct=1 weightb=1 open_gop=0 weightp=2 keyint=250 keyint_min=25 scenecut=40 intra_refresh=0 rc_lookahead=40 rc=crf mbtree=1 crf=18.0 qcomp=0.60 qpmin=0 qpmax=69 qpstep=4 ip_ratio=1.40 aq=1:1.00
Output #0, mp4, to 'renders\rep\minimap.mp4':
  Metadata:
    encoder         : Lavf58.3.100
    Stream #0:0: Video: h264 (libx264) (avc1 / 0x31637661), yuv420p, 500x650, q=-1--1, 30 fps, 15360 tbn, 30 tbc
    Metadata:
      encoder         : Lavc58.8.100 libx264
    Side data:
      cpb: bitrate max/min/avg: 0/0/0 buffer size: 0 vbv_delay: -1
```

**Comment by enzanki-ars** [comment]
I am looking into the issue on a fresh windows install.

**Comment by enzanki-ars** [comment]
Let's see if #36 will fix this issue potentially.

**Comment by enzanki-ars** [comment]
I have just released `v1.4.0-alpha1` at https://github.com/enzanki-ars/rocket-league-replay-analysis/releases/tag/v1.4.0-alpha1, though it is missing the minimap output.  Mind testing to see if that fixes the issues?
